WordPress/WordPressTest/ViewRelated/Post/Controllers/PreviewWebKitViewControllerTests.swift

```diff
@@ -0,0 +1,82 @@
+//
+//  PreviewWebKitViewController.swift
+//  WordPressTest
+//
+//  Created by Brandon Titus on 1/13/20.
+//  Copyright © 2020 WordPress. All rights reserved.
+//
+
+import XCTest
+@testable import WordPress
+
+class PreviewWebKitViewControllerTests: XCTestCase {
+
+    private var rootWindow: UIWindow!
+    private var context: NSManagedObjectContext!
+    private var navController = UINavigationController()
+
+    override func setUp() {
+        super.setUp()
+        context = TestContextManager().mainContext
+
+        rootWindow = UIWindow(frame: UIScreen.main.bounds)
+        rootWindow.isHidden = false
+        rootWindow.rootViewController = navController
+    }
+
+    override func tearDown() {
+        super.tearDown()
+        context = nil
+        ContextManager.overrideSharedInstance(nil)
+    }
+
+    func testMissingPermalink() {
+        let post = PostBuilder(context).drafted().build()
+
+        let vc = PreviewWebKitViewController(post: post, previewURL: nil)
+        XCTAssertEqual(vc.url.absoluteString, "about:blank", "Should load blank page when no permalink is available")
+    }
+
+    func testDraftToolbarItems() {
+
+        let post = PostBuilder(context).drafted().build()
+        post.permaLink = "http://example.com/"
+
+        let vc = PreviewWebKitViewController(post: post, previewURL: nil)
+        let items = vc.toolbarItems(linkBehavior: vc.linkBehavior)
+
+        XCTAssertTrue(items.contains(vc.publishButton), "Should contain publish button")
+        XCTAssertFalse(items.contains(vc.safariButton),
+                       "Should contain back button")
```

Comment:
Most of these tests say `Should contain back button`. They probably should be specific to the button type.